src/collective/fingerpointing/subscribers/registry_logger.py

```diff
@@ -4,28 +4,30 @@
 from collective.fingerpointing.logger import log_info
 from collective.fingerpointing.utils import get_request_information
 from plone import api
-from plone.api.exc import InvalidParameterError
 from plone.registry.interfaces import IRecordModifiedEvent
 
 
+def safe_utf8(s):
+    if isinstance(s, unicode):
```

Comment:
@jensens can you help me here? how can I make this Python 3 compatible?

Reply:
I would use ``six`` like so:
```
import six 

if isinstance(s, six.text_type):
```
This introduces a dependency on six, but in fact Plone 5 already depends on it (I am not sure for Plone 4.3, but it should not hurt).
See also https://pythonhosted.org/six/#six.text_type